app/helpers/error_helper.rb

```diff
@@ -122,9 +122,11 @@ class ErrorAggregator
         headers: ["Row #", "Sample Name"],
         title: ->(num_rows, _) { "#{num_rows} rows are missing host genomes." },
       },
+      # TODO: (gdingle): This behavior will change after removal of admin-only of new host genome input.
```

Comment:
add date?

Reply:
I think the link to the JIRA is sufficient.